stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Channel.kt

```diff
@@ -32,7 +32,7 @@ internal fun Channel.diff(other: Channel): ChannelListPayloadDiff {
         usersChanged = usersChanged,
         readStateChanged = read != other.read,
         lastMessageChanged = getLastMessage() != other.getLastMessage(),
-        unreadCountChanged = unreadCount != other.unreadCount,
+        unreadCountChanged = unreadCount != other.unreadCount && other.unreadCount != null,
```

Comment:
I wonder if `null` can be a valid value when there is no `unreadCount`

Reply:
I tested a few scenarios including reading the channel from another device while the same user was logged in on two devices and saw no adverse effects.